test/import/FSHErrorListener.test.ts

```diff
@@ -5,11 +5,12 @@ describe('FSHErrorListener', () => {
   beforeEach(() => loggerSpy.reset());
 
   it('should not log any errors for valid FSH', () => {
-    const input = `
+    /*
     Profile: MyPatient
     Parent: Patient
     * active = true
-    `;
+    */
+    const input = 'Profile: MyPatient\nParent: Patient\n* active = true';
```

Comment:
I think it might be useful to build the test inputs by joining strings together with `os.EOL`. That way, we can be sure that things work as expected with different line endings. It will also make it a little easier to read what the input is without having to include the comment before the input. I realize this is asking for a lot of modifications, but maybe a search/replace will get most of them?

Also, most of the test cases for FSHImporter that aren't modified here are raising the left-alignment error on line 2015, but they still pass since only one of the test files uses context paths for rules. But, it might still be good to fix these as well? I can imagine problems/annoyances arising due to getting an extra error in the logger for each rule. But, I also recognize that this is a lot of extra spots to fix.

Reply:
I think I agree w/ @mint-thompson .  These are kind of difficult to read as single-liners, and while the comments help, it's also easy for those to come out of sync w/ the real code.  I feel like it would be easier to read as:
```
const input = [
  'Profile: MyPatient',
  'Parent: Patient',
  '* active = true'
].join(os.EOL);
```

Another approach would be to define a function that takes a string and normalizes out the leading spaces.  E.g.:
```
const input = leftAlign(`
    Profile: MyPatient
    Parent: Patient
    * active = true
    `);
```
The `leftAlign` function would find the number of spaces to the first non-space character (in the first non-blank line) and then remove that number of spaces from all lines (similar to our implementation of `"""`).

Like Mint, I feel bad because this is likely a lot of (tedious) work.  If these are the only changes you made in these test files, though, it might be easier just to revert the files to the original versions and edit those.

Reply:
No this is totally reasonable! I'll update the test cases to make use of `os.EOL` and also make the unit test changes for the rest of the tests in the FSHImporter test suite as well.

Reply:
I agree with the both of you, I kind of struggled with finding the most intuitive way to make these changes. I'll update the FSHImporter test suite so that they're a bit more readable and consistent, and also use `os.EOL` instead of the explicit newline character.